src/webapp/pages/Home.js

```diff
@@ -1,7 +1,98 @@
 import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+import LocalGroceryStoreIcon from "@material-ui/icons/LocalGroceryStore";
+import MapIcon from "@material-ui/icons/Map";
+import { Link } from "@reach/router";
+import { useTranslation } from "react-i18next";
+import sharedStylesFn from "../style/sharedStyles";
 
-import JustTextContent from "webapp/components/JustTextContext";
+const useStyles = makeStyles((theme) => ({
+  ...sharedStylesFn(theme),
+
+  icon: {
+    marginRight: "0.5rem",
+  },
+  subAppPaperLink: {
+    fontSize: "1.5rem",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+    "&:hover": {
+      backgroundColor: "hsla(0, 0%, 0%, 0.05)",
```

Comment:
minor nit: i think the bg color can be a _tad_ darker for better contrast on brighter screens.

Reply:
@piratefsh you want a darker colour when hovering over the Papers that link to the other apps? Or do you mean the actual bg colour outside of the Papers?

Reply:
ah I meant the actual BG outside of the Paper! maybe increase the opacity on backgroundColor here. feel free to tweak to see what looks good.

Reply:
Modifying the background color would require [affecting the root](https://stackoverflow.com/questions/48186478/custom-theme-background-color) styling which would affect each page as far as I can tell, so as a compromise I've increased the `elevation` used in the Papers

![image](https://user-images.githubusercontent.com/7434338/97639442-ed6ddb80-1a14-11eb-969f-901c8d1a2bd9.png)